pkg/resource/deploy/source_eval.go

```diff
@@ -1008,6 +1021,76 @@ func inheritFromParent(child resource.Goal, parent resource.Goal) *resource.Goal
 	return &goal
 }
 
+type sourcePositions struct {
+	projectRoot string
+}
+
+func newSourcePositions(projectRoot string) *sourcePositions {
+	if projectRoot == "" {
+		projectRoot = "/"
+	} else {
+		contract.Assertf(filepath.IsAbs(projectRoot), "projectRoot is not an absolute path")
+		projectRoot = filepath.Clean(projectRoot)
+	}
+	return &sourcePositions{projectRoot: projectRoot}
+}
+
+// source position URLs:
+// scheme://path#line[,col]
+func (s *sourcePositions) parseSourcePosition(raw string) (string, error) {
+	posURL, err := url.Parse(raw)
+	if err != nil {
+		return "", err
+	}
+	if posURL.Scheme != "file" {
+		return "", fmt.Errorf("unrecognized scheme %q", posURL.Scheme)
+	}
+	line, col, _ := strings.Cut(posURL.Fragment, ",")
+	if line == "" {
+		return "", fmt.Errorf("missing line number")
+	}
+	if _, err := strconv.ParseInt(line, 10, 0); err != nil {
+		return "", fmt.Errorf("invalid line number %q: %w", line, err)
+	}
+	if col != "" {
+		if _, err := strconv.ParseInt(col, 10, 0); err != nil {
+			return "", fmt.Errorf("invalid column number %q: %w", col, err)
+		}
+	}
+
+	file := filepath.FromSlash(posURL.Path)
+	if !filepath.IsAbs(file) {
+		return "", fmt.Errorf("source positions must include absolute paths")
+	}
+	rel, err := filepath.Rel(s.projectRoot, file)
+	if err != nil {
+		return "", fmt.Errorf("making relative path: %w", err)
+	}
+
+	logging.V(5).Infof("(%q, %q) -> %q", file, s.projectRoot, rel)
```

Comment:
There's no other log lines to give this context that it's a file remapping, can that be added to the log text.

Reply:
Ah yeah this can just be removed, frankly. Used this for debugging.